tests/kvbm_integration/test_consolidator_config_unit.py

```
@@ -0,0 +1,78 @@
+    def test_production_build_kv_connector_config(self):
+        """Assert build_kv_connector_config output works with consolidator check.
+
+        This is the key regression test: if llm_worker.py changes the type
+        returned by build_kv_connector_config, this test will catch it.
+        """
+        pytest.importorskip("tensorrt_llm", reason="tensorrt_llm not installed")
+        from dynamo.trtllm.workers.llm_worker import build_kv_connector_config
+
+        # Mock the Config object with connector=["kvbm"]
+        class FakeConfig:
+            connector = ["kvbm"]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)